docs/source/openshift/openshift-client-binary.adoc

```diff
@@ -27,26 +28,26 @@ export PATH="/Users/john/.minishift/cache/oc/v1.5.0:$PATH"
 ----
 
 [[minishift-oc-context]]
-== Minishift oc context
+== Minishift oc Context
```

Comment:
> We're supposed to use title caps in all headings, according to the openshift guidelines

I know, however, in this context it is a bit unfortunate that 'oc' is lowercase. Never mind.

> In fact, I'd like to suggest that we ditch the term "oc context", now that I dug a little into it and you also confirmed that it's a CLI Profile, and I would propose that we rename it as such so that we're aligned with the terms.

I am happy to rename the section "Minishift CLI Profile", however, we also need to keep the term 'context'. The term is correct as well and probably closer to what actually happens on a technical level - https://docs.openshift.org/latest/cli_reference/manage_cli_profiles.html

So +1 for changing the title, -1 for removing the mention to 'context'.